BuildResidentialHPXML/measure.rb

```diff
@@ -0,0 +1,3472 @@
+    arg.setDefaultValue('EnergyFactor')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeStringArgument('water_heater_efficiency', true)
+    arg.setDisplayName('Water Heater: Efficiency')
+    arg.setDescription('EnergyFactor=Ratio of useful energy output from water heater to the total amount of energy delivered from the water heater. UniformEnergyFactor=The uniform energy factor of water heater.')
+    arg.setDefaultValue(Constants.Auto)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('water_heater_recovery_efficiency', true)
+    arg.setDisplayName('Water Heater: Recovery Efficiency')
+    arg.setDescription('Ratio of energy delivered to water heater to the energy content of the fuel consumed by the water heater. Only used for non-electric water heaters.')
+    arg.setUnits('Frac')
+    arg.setDefaultValue(0.76)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('water_heater_standby_loss', true)
+    arg.setDisplayName('Water Heater: Standby Loss')
+    arg.setDescription('The standby loss of water heater.')
+    arg.setUnits('Frac')
+    arg.setDefaultValue(0)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('water_heater_jacket_rvalue', true)
+    arg.setDisplayName('Water Heater: Jacket R-value')
+    arg.setDescription('The jacket R-value of water heater.')
+    arg.setUnits('h-ft^2-R/Btu')
+    arg.setDefaultValue(0)
+    args << arg
+
+    hot_water_distribution_system_type_choices = OpenStudio::StringVector.new
+    hot_water_distribution_system_type_choices << HPXML::DHWDistTypeStandard
+    hot_water_distribution_system_type_choices << HPXML::DHWDistTypeRecirc
+
+    arg = OpenStudio::Measure::OSArgument::makeChoiceArgument('hot_water_distribution_system_type', hot_water_distribution_system_type_choices, true)
+    arg.setDisplayName('Hot Water Distribution: System Type')
+    arg.setDescription('The type of the hot water distribution system.')
+    arg.setDefaultValue(HPXML::DHWDistTypeStandard)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('standard_piping_length', true)
+    arg.setDisplayName('Hot Water Distribution: Standard Piping Length')
+    arg.setUnits('ft')
+    arg.setDescription('The length of the standard piping.')
+    arg.setDefaultValue(50)
+    args << arg
+
+    recirculation_control_type_choices = OpenStudio::StringVector.new
+    recirculation_control_type_choices << HPXML::DHWRecirControlTypeNone
+    recirculation_control_type_choices << HPXML::DHWRecirControlTypeTimer
+    recirculation_control_type_choices << HPXML::DHWRecirControlTypeTemperature
+    recirculation_control_type_choices << HPXML::DHWRecirControlTypeSensor
+    recirculation_control_type_choices << HPXML::DHWRecirControlTypeManual
+
+    arg = OpenStudio::Measure::OSArgument::makeChoiceArgument('recirculation_control_type', recirculation_control_type_choices, true)
+    arg.set
```

Comment:
And same here.

Reply:
Which are you referring to here?

Reply:
backup_heating_switchover_temp: 25.0

Reply:
I decided to expose `heat_pump_backup_heating_switchover_temp`.